File: JavaScript/lv1/콜라츠 추측.js
function solution(n) {
  let result = 0;

  while (n !== 1) {
    if (n % 2 === 0) {
      n /= 2;
      result++;
    } else {
      n = 3 * n + 1;
      result++;
    }
  }

  if (result > 500) {
    return -1;
  } else {
    return result;
  }
}

File: Python/lv1/콜라츠 추측.py
def solution(num):
    count = 0
    while num != 1:
        if num % 2 == 0:
            num /= 2
        else:
            num = num * 3 + 1
        count += 1
        if count == 500:  # 만약 500번 이상 반복해도 1이 되지 않는다면 -1을 반환
            return -1
    return count
